pkgs/development/python-modules/markdownify/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytestCheckHook
+, flake8
+, beautifulsoup4
+, six }:
+buildPythonPackage rec {
+  pname = "markdownify";
+  version = "0.11.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-1hP5WyZJ2Lg8TEyy5sJM6ox99OB9QY+S+iWtDduwReA=";
+  };
+
+  nativeBuildInputs = [ flake8 ];
```

Comment:
Seems like the flake8 dependency is only needed for their lint command, which we don't use. Please remove it, as it reduces the closure size.

Reply:
I filed https://github.com/matthewwithanm/python-markdownify/issues/70 and added a downstream patch in the mean time.

Reply:
Upstream removed the build requirement.